adminSiteClient/EditorTextTab.tsx

```diff
@@ -88,13 +106,19 @@ export class EditorTextTab extends React.Component<{ editor: ChartEditor }> {
                         softCharacterLimit={100}
                     />
                     <Toggle
-                        label="Hide automatic time/entity"
-                        value={!!grapher.hideTitleAnnotation}
-                        onValue={action(
-                            (value: boolean) =>
-                                (grapher.hideTitleAnnotation =
-                                    value || undefined)
-                        )}
+                        label="Hide automatic entity (where possible)"
+                        value={!!grapher.hideTitleAnnotation?.entity}
+                        onValue={this.onToggleTitleAnnotationEntity}
+                    />
+                    <Toggle
+                        label="Hide automatic time (where possible)"
+                        value={!!grapher.hideTitleAnnotation?.time}
+                        onValue={this.onToggleTitleAnnotationTime}
+                    />
+                    <Toggle
+                        label="Don't prepend 'Change in' automatically (where possible)"
```

Comment:
Since this is the most obscure one (it's only relevant for a small subset of charts), I would try to make its name clearer:
`Don't prepend 'Change in' in relative line charts`.

Also, we sometimes hide controls that are not relevant for the current chart.
In this case, whether it's relevant depends on `grapher.isLineChart && (grapher.isRelativeMode || grapher.canToggleRelativeMode)`.